### PR TITLE
Add chapter index

### PR DESCRIPTION
**Pull request by joaofl**
I implemented the chapter numbering, so that chapters folders show in order. I made some other overall polishing (please check the commits description).

I guess you may want to accept/reject some of my suggestions, so feel free to discuss them.

### CODE REVIEW

**Comment by knowbee** [comment]
> I implemented the chapter numbering, so that chapters folders show in order. I made some other overall polishing (please check the commits description).
> 
> I guess you may want to accept/reject some of my suggestions, so feel free to discuss them.

Thanks  for your contribution, I think you did a great job.Have you tested this PR on Windows OS?

**Comment by joaofl** [comment]
> > I implemented the chapter numbering, so that chapters folders show in order. I made some other overall polishing (please check the commits description).
> > I guess you may want to accept/reject some of my suggestions, so feel free to discuss them.
> 
> Thanks for your contribution, I think you did a great job.Have you tested this PR on Windows OS?

Good that It makes sense to you. I did not have the chance to test it on windows yet. Do you have a test setup that you use to validate your changes?

**Comment by knowbee** [comment]
> > > I implemented the chapter numbering, so that chapters folders show in order. I made some other overall polishing (please check the commits description).
> > > I guess you may want to accept/reject some of my suggestions, so feel free to discuss them.
> > 
> > 
> > Thanks for your contribution, I think you did a great job.Have you tested this PR on Windows OS?
> 
> Good that It makes sense to you. I did not have the chance to test it on windows yet. Do you have a test setup that you use to validate your changes?

I don't have a test setup

**Comment by knowbee** [comment]
> > > I implemented the chapter numbering, so that chapters folders show in order. I made some other overall polishing (please check the commits description).
> > > I guess you may want to accept/reject some of my suggestions, so feel free to discuss them.
> > 
> > 
> > Thanks for your contribution, I think you did a great job.Have you tested this PR on Windows OS?
> 
> Good that It makes sense to you. I did not have the chance to test it on windows yet. Do you have a test setup that you use to validate your changes?

I don't have a test setup

File: llvd/app.py [hunk 1]
```diff
@@ -139,40 +137,36 @@ def download_entire_course(self):
...
+                        for file in os.listdir(chapter_path):
                             if file.endswith(".mp4") and "-" in file:
                                 ff = re.split(
                                     "\d+-", file)[1].replace(".mp4", "")
                                 current_files.append(ff)
                     except Exception as e:
                         if 'url' in str(e):
                             click.echo(
-                                click.style(f"This video is locked, you probably need a premium account", fg="red"))
+                                click.style(f"This video is locked, you probably"\
```
Comment (knowbee) [inline llvd/app.py:151]: Add a space between `probably` and `need`

Comment (joaofl) [reply llvd/app.py]: Done

**Review by knowbee — APPROVED** [review]

**Comment by joaofl** [comment]
Did you test it on windows? I did not test yet

**Comment by joaofl** [comment]
@knowbee I believe you are the one supposed to merge it.

**Comment by knowbee** [comment]
> Did you test it on windows? I did not test yet

I did not find a way to do so, but if it breaks we shall fixed that

**Comment by knowbee** [comment]
> @knowbee I believe you are the one supposed to merge it.

Yeah..Thanks for the contribution

**Pull request merged**